go.mod

```diff
@@ -12,19 +12,32 @@ require (
 	cloud.google.com/go v0.57.0
 	cloud.google.com/go/pubsub v1.3.1
 	cloud.google.com/go/storage v1.6.0
-	github.com/crossplane/crossplane v0.13.0
 	github.com/crossplane/crossplane-runtime v0.10.0
 	github.com/crossplane/crossplane-tools v0.0.0-20201007233256-88b291e145bb
+	github.com/go-logr/zapr v0.1.1 // indirect
 	github.com/google/go-cmp v0.5.0
 	github.com/googleapis/gax-go v1.0.3
 	github.com/googleapis/gax-go/v2 v2.0.5
 	github.com/imdario/mergo v0.3.9
+	github.com/kr/text v0.2.0 // indirect
 	github.com/mitchellh/copystructure v1.0.0
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.1.0 // indirect
+	github.com/spf13/cobra v1.0.0 // indirect
+	github.com/stretchr/testify v1.5.1 // indirect
+	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a // indirect
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
+	golang.org/x/sys v0.0.0-20200831180312-196b9ba8737a // indirect
+	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
+	golang.org/x/tools v0.0.0-20200916195026-c9a70fc28ce3 // indirect
```

Comment:
Some of these changes feel surprising to me, but I see CI is passing 🤷‍♂️

Reply:
Same - I made sure `gopls` wasn't running, ran `go mod tidy`, etc and they still stuck around. 🤷